phpunit.xml

```diff
@@ -15,27 +15,6 @@
             <directory>packages-tests</directory>
             <directory>utils-tests</directory>
         </testsuite>
-
-        <!-- divide the rules-tests roughly in 3 equal sized groups -->
```

Comment:
as the one person who introduced this test-suite splitting, I am really happy that the suite is so much faster now, so we don't need the splitting.

great job!